File: benchmarkDS/Vanilla/CLAUDE_claude-sonnet-4-5-20250929/Lse12.py

def getValueFromArray(array, size, index):
    if 0 <= index < size:
        return array[index]
    return -1  # Return -1 for invalid index

def main():
    # Test case 1
    arr1 = [10, 20, 30, 40, 50]
    print("Test 1:", getValueFromArray(arr1, 5, 2))  # Expected: 30
    
    # Test case 2
    arr2 = [5, 15, 25, 35, 45]
    print("Test 2:", getValueFromArray(arr2, 5, 0))  # Expected: 5
    
    # Test case 3
    arr3 = [100, 200, 300, 400, 500]
    print("Test 3:", getValueFromArray(arr3, 5, 4))  # Expected: 500
    
    # Test case 4
    arr4 = [7, 14, 21, 28, 35]
    print("Test 4:", getValueFromArray(arr4, 5, 3))  # Expected: 28
    
    # Test case 5
    arr5 = [1, 2, 3, 4, 5]
    print("Test 5:", getValueFromArray(arr5, 5, 1))  # Expected: 2
    
    # If you want to read from standard input and use as exit code:
    # import sys
    # index = int(input())
    # result = getValueFromArray(arr1, 5, index)
    # sys.exit(result)

if __name__ == "__main__":
    main()

File: benchmarkDS/Vanilla/CLAUDE_claude-sonnet-4-5-20250929/Lse12.cpp

#include <iostream>
using namespace std;

int getValueFromArray(int array[], int size, int index) {
    if (index >= 0 && index < size) {
        return array[index];
    }
    return -1; // Return -1 for invalid index
}

int main() {
    // Test case 1
    int arr1[] = {10, 20, 30, 40, 50};
    cout << "Test 1: " << getValueFromArray(arr1, 5, 2) << endl; // Expected: 30
    
    // Test case 2
    int arr2[] = {5, 15, 25, 35, 45};
    cout << "Test 2: " << getValueFromArray(arr2, 5, 0) << endl; // Expected: 5
    
    // Test case 3
    int arr3[] = {100, 200, 300, 400, 500};
    cout << "Test 3: " << getValueFromArray(arr3, 5, 4) << endl; // Expected: 500
    
    // Test case 4
    int arr4[] = {7, 14, 21, 28, 35};
    cout << "Test 4: " << getValueFromArray(arr4, 5, 3) << endl; // Expected: 28
    
    // Test case 5
    int arr5[] = {1, 2, 3, 4, 5};
    cout << "Test 5: " << getValueFromArray(arr5, 5, 1) << endl; // Expected: 2
    
    // If you want to read from standard input and use as exit code:
    // int index;
    // cin >> index;
    // int result = getValueFromArray(arr1, 5, index);
    // return result;
    
    return 0;
}
